android/src/main/java/com/swmansion/rnscreens/ScreenContainer.java

```diff
@@ -352,14 +356,16 @@ protected void performUpdate() {
       }
     }
 
-    // detect if we are "transitioning" based on the number of active screens
-    int activeScreens = 0;
+    boolean transitioning = true;
+
     for (int i = 0, size = mScreenFragments.size(); i < size; i++) {
-      if (isScreenActive(mScreenFragments.get(i))) {
-        activeScreens += 1;
+      ScreenFragment screenFragment = mScreenFragments.get(i);
+      int active = getScreenActive(screenFragment);
+      if (active == 2) {
```

Comment:
can we define a constant for this and name it appropriately such that this code is easier to follow